homeassistant/components/bayesian/strings.json

```
@@ -14,5 +14,192 @@
       "name": "[%key:common::action::reload%]",
       "description": "Reloads Bayesian sensors from the YAML-configuration."
     }
+  },
+  "options": {
+    "error": {
+      "extreme_prior_error": "[%key:component::bayesian::config::error::extreme_prior_error%]",
+      "extreme_threshold_error": "[%key:component::bayesian::config::error::extreme_threshold_error%]",
+      "equal_probabilities": "[%key:component::bayesian::config::error::equal_probabilities%]",
+      "extreme_prob_given_error": "[%key:component::bayesian::config::error::extreme_prob_given_error%]"
+    },
+    "step": {
+      "init": {
+        "title": "Sensor options",
+        "description": "These options affect how much evidence is required for the Bayesian sensor to be considered 'on'.",
+        "data": {
+          "probability_threshold": "[%key:component::bayesian::config::step::user::data::probability_threshold%]",
+          "prior": "[%key:component::bayesian::config::step::user::data::prior%]",
+          "device_class": "[%key:component::bayesian::config::step::user::data::device_class%]",
+          "name": "[%key:common::config_flow::data::name%]"
+        },
+        "data_description": {
+          "probability_threshold": "[%key:component::bayesian::config::step::user::data_description::probability_threshold%]",
+          "prior": "[%key:component::bayesian::config::step::user::data_description::prior%]"
+        }
+      }
+    }
+  },
+  "config": {
+    "error": {
+      "extreme_prior_error": "'Prior' set to 0% means that it is impossible for the sensor to show 'on' and 100% means it will never show 'off', use a close number like 0.1% or 99.9% instead",
+      "extreme_threshold_error": "'Probability threshold' set to 0% means that the sensor will always be 'on' and 100% mean it will always be 'off', use a close number like 0.1% or 99.9% instead",
+      "equal_probabilities": "If 'Probability given true' and 'Probability given false' are equal, this observation can have no effect, and is therefore redundant",
+      "extreme_prob_given_error": "If either 'Probability given false' or 'Probability given true' is 0 or 100 this will create certainties that override all other observations, use numbers close to 0 or 100 instead"
+    },
+    "step": {
+      "user": {
+        "title": "Add a Bayesian sensor",
+        "description": "Create a binary sensor which observes the state of multiple sensors to estimate whether an event is occurring, or if something is true. See [the documentation](https://www.home-assistant.io/integrations/Bayesian/) for more details.",
```

It's suggested to define (documentation) URLs as constants so they can be changed at any time, even when translations lag behind. We don't have those fixed in existing code everywhere, but for new additions it makes sense.

---

I'm not a programmer so I can't comment on how to implement this in your case. Then leave it as is and just change the URL to all lowercase for consistency with the links inside the website:

```suggestion
        "description": "Create a binary sensor which observes the state of multiple sensors to estimate whether an event is occurring, or if something is true. See [the documentation](https://www.home-assistant.io/integrations/bayesian/) for more details.",
```

---

Thank you @NoRi2909.
Unfortunately I've made some more changes to `strings.json` today to improve clarity of errors and some of the hint text. Do have a look if time allows.

---

You can have a look at https://github.com/home-assistant/core/pull/147015/files regarding the URL definition.

It adds

```
description_placeholders: dict[str, str] = {
  "api_key_url": "https://developers.google.com/maps/documentation/weather/get-api-key",
}
```

to `__init__.py` and then uses 

```
"description": "Get your API key from [here]({api_key_url})."
```

in `strings.json` - perhaps this works for you here, too.